app-devel/gcc-13/autobuild/prepare

```diff
@@ -0,0 +1,29 @@
+abinfo "Nutering fixinc.sh in gcc/Makefile ..."
+sed -i 's@\./fixinc\.sh@-c true@' "$SRCDIR"/gcc/Makefile.in
+
+abinfo 'Adjusting $CPPFLAGS ...'
+sed -i "/ac_cpp=/s/\$CPPFLAGS/\$CPPFLAGS -O2/" "$SRCDIR"/{libiberty,gcc}/configure
+
+abinfo "Adjusting path for Go importing ..."
+sed -i 's@%D@-L/usr/lib@g' "$SRCDIR"/gcc/go/lang-specs.h
```

Comment:
Best to make all sed operations patches - I know this is not yet a thing in Core, but best to handle it more cleanly.